pkg/compact/compact.go

```diff
@@ -440,23 +442,23 @@ func (cg *Group) Resolution() int64 {
 
 // Compact plans and runs a single compaction against the group. The compacted result
 // is uploaded into the bucket the blocks were retrieved from.
-func (cg *Group) Compact(ctx context.Context, dir string, comp tsdb.Compactor) (ulid.ULID, error) {
+func (cg *Group) Compact(ctx context.Context, dir string, comp tsdb.Compactor) (bool, error) {
 	subDir := path.Join(dir, cg.Key())
 
 	if err := os.RemoveAll(subDir); err != nil {
-		return ulid.ULID{}, errors.Wrap(err, "clean compaction group dir")
+		return false, errors.Wrap(err, "clean compaction group dir")
 	}
 	if err := os.MkdirAll(subDir, 0777); err != nil {
-		return ulid.ULID{}, errors.Wrap(err, "create compaction group dir")
+		return false, errors.Wrap(err, "create compaction group dir")
 	}
 
-	id, err := cg.compact(ctx, subDir, comp)
+	continueCompact, err := cg.compact(ctx, subDir, comp)
 	if err != nil {
 		cg.compactionFailures.Inc()
 	}
 	cg.compactions.Inc()
 
-	return id, err
+	return continueCompact, err
```

Comment:
Abit confused here ... `continueCompaction` here turns into `nothingToCompact` in `thanos/compact.go` and in the inner `compact` below you return `nothingToCompact` as `true` if its done ... so this then gets assinged to `continueCompact` ... maybe im missing something but the vars are abit confusing.

Reply:
Can we just change it all to `continueCompact`? Think this would read a lot better and help keep it simple.